cfgov/jinja2/v1/_includes/organisms/filterable-list.html

```diff
@@ -181,9 +185,14 @@
                         </div>
                     </div>
                 {% else %}
-                    {% import 'organisms/post-preview.html' as post_preview with context %}
+                    {% import 'organisms/post-preview.html' as post_preview without context %}
                     {% for post in posts %}
-                        {{ post_preview.render(post, value) }}
+                        {{ post_preview.render(
+                            post,
+                            value,
+                            show_date=show_post_dates|default(true),
+                            show_tags=show_post_tags|default(true)
```

Comment:
Trivial and we don't adhere to it, but spacing out filters can make them stand out a bit more `show_tags=show_post_tags | default(true)`